cassiopeia/simulator/Cas9LineageTracingDataSimulator.py

```diff
@@ -73,15 +73,17 @@ class Cas9LineageTracingDataSimulator(LineageTracingDataSimulator):
         size_of_cassette: Number of editable target sites per cassette
         mutation_rate: Exponential parameter for the Cas9 cutting rate. The user
             can either pass in a float, in which every site will mutate at
-            the same rate, or a list with an entry for each cut site indicating
-            potentially a different mutation rate for every site.
+            the same rate, or a list a list of floats of length 
+            `size_of_cassette` or `number_of_cassettes * size_of_cassette`.
         state_generating_distribution: Distribution from which to simulate state
             likelihoods. This is only used if mutation priors are not
             specified to the simulator.
         number_of_states: Number of states to simulate
-        mutation_priors: A mapping from state to probability that a user
-            can specify. If this argument is not None, states will not be
-            pulled from the state distribution.
+        mutation_priors: An optional dictionary mapping states to their prior 
+            probabilities. Can also be a list of dictionaries of length 
+            `size_of_cassette` or `number_of_cassettes * size_of_cassette`.
```

Comment:
same request as above - make it more clear what the expected behavior is for each of these parameter styles.